docker/community/docker-compose.yml

```diff
@@ -66,11 +81,20 @@ services:
       IS_MULTI_TENANCY_ENABLED: ${IS_MULTI_TENANCY_ENABLED}
     ports:
       - '3000:3000'
+    healthcheck:
+      test: ["CMD-SHELL", "wget --spider --quiet http://localhost:3000/v1/health-check || exit 1"]
```

Comment:
@twentyone24 I see you have hardcoded value l`ocalhost:3000`. api service can run on different port, IP or custom domain in case of self hosting.

Reply:
@jainpawan21 appreciate your comments, 
- regarding `HOST`, the `localhost`, `api` hosts should  still work even  with custom configuration or with reverse proxy, as these HEALTHCHECKs execute within the container under the same network. 
- regarding `PORTS`, I've hardcoded them as the existing config has hardcoded ports setup `'3000:3000'` unless this is not changed that won't be affected. and better way to do this would be to decouple into an .env and pass them as arguments wherever required. 

hope this resolves your concerns.